### PR TITLE
Added OEMDRV label.

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Describe the overall goals of this pull request.

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/ay_auto_installation.xml [hunk 1]
```diff
@@ -92,6 +92,10 @@ xml:id="Invoking">
    <literal>&ay;=<replaceable>URL</replaceable></literal>.
   </para>
 
+  <para>
+    Alternatively, you can place the <filename>autoinst.xml</filename> to a device, mounted either physically or virtually, that is labeled <literal>OEMDRV</literal>. In this case, you do not need to specify the location of the <filename>autoinst.xml</filename> on the kernel command line. The <literal>autoinst.xml</literal> must be located in the root direcotry of the device.
```
Comment (imobachgs) [inline xml/ay_auto_installation.xml:96]: typo: direcotry

**Review by imobachgs — COMMENTED** [review]
The text looks good. 

Thank you!

**Review by imobachgs — APPROVED** [review]

File: xml/ay_auto_installation.xml (hunk 1 above)
Comment (dariavladykina) [inline xml/ay_auto_installation.xml:96]:
```suggestion
    Alternatively, you can place the <filename>autoinst.xml</filename> to a device, mounted either physically or virtually, that is labeled <literal>OEMDRV</literal>. In this case, you do not need to specify the location of the <filename>autoinst.xml</filename> on the kernel command line. The <literal>autoinst.xml</literal> must be located in the root directory of the device.
```

**Review by dariavladykina — COMMENTED** [review]
Just a typo here.

**Pull request merged**